### DISCUSSION TITLE
Sample loop end point!

### DISCUSSION THREAD

**Discussion by ptowns** [bangcorrupt/hacktribe#58] [Ideas]
So one of the biggest mysteries to me has always been the edit knob on the sampler. Hard to find a good use for it. Basically feels like a waste of a knob. Now if it could modulate the start AND END point together! Instant wavetable scan. In another post I talked about placing single cycle waves in an order to use the osc knob to scan through a 'wave table' but the edit knob would alow a true modulatable wavetable if you could shift the loop around the sample. Of coarse in a dream world let me press shit and turn the edit knob to change the end point of the sample(sample size), now we can do some granular fun stuff.  I know the electribe 2 is capable of some granular stuff just a bit outside of the box. 

https://www.instagram.com/s/aGlnaGxpZ2h0OjE3ODU0MDQzMjEzNDQ3NjU1?story_media_id=2519405378924765265_1337820985&utm_medium=copy_link

**Comment by ptowns** [comment]
Also seq play under trigger mode is a mystery! I get what it is, just not sure why or how someone would make use of it.

**Reply by bangcorrupt** [reply]
One of the original promo videos was a great demo of Seq Play, I can't find it now. By the way, I can't see the instagram links you post, I think it requires an account.

**Reply by ptowns** [reply]
I remember that demo! Everyone in the comments were saying how is he doing that. But I was over here thinking why! Not sure i have seen anyone else use it. Essentially one could resample a sequence and then play the sample in trigger mode making the same thing. Sure you COULD modify the part still using seq play, but with only 2 hands it will get difficult fast!

**Reply by bangcorrupt** [reply]
It would be more useful if it could latch.  Combined with Last Step and Groove Templates it could make make some really wonky polyrhythms.

**Reply by listenForward** [reply]
I absolutely LOVE the idea of having a Trigger Mode option to **_Seq Latch_**

It helped inspire me to start [a new Discussion for brainstorming on Trigger Modes.](https://github.com/bangcorrupt/hacktribe/discussions/203)

**Comment by bangcorrupt** [comment]
I'd like to make sample loop modes similar to how they work on Polyend Tracker.

**Comment by juanjo1138** [comment]
Granular!

**Reply by ptowns** [reply]
Have been meaning to come back to this and post the idea. 

https://user-images.githubusercontent.com/65518342/165875854-4fd253a7-b0b5-443e-8e29-cdde35fa2cdb.mp4

Basically if the loop and end point could move WITH the edit know you could have a 64 sample morphing wavetable.
